ApiDoctor.Validation/ExtensionMethods.cs

```diff
@@ -225,13 +220,25 @@ public static bool TryGetProperty(this JContainer container, string propertyName
         }
 
         /// <summary>
-        /// Only removes markdown links with the format [text](link_target) or [text]: link_reference
+        /// Removes markdown styling from text i.e. links, bold, italics, code fence 
         /// </summary>
-        /// <param name="text"></param>
-        /// <returns></returns>
-        public static string StripMarkdownLinks(this string text)
+        /// <param name="markdownText">String with possible markdown style</param>
+        /// <returns>Text without markdown styling</returns>
+        public static string RemoveMarkdownStyling(this string markdownText)
         {
-            return Regex.Replace(text, markdownLinkRegex.ToString(), "$1");
+            // Remove bold (**text**)
+            markdownText = Regex.Replace(markdownText, @"\*\*(.*?)\*\*", "$1");
```

Comment:
Can we keep/define a compiled regex instances, e.g.

```cs
private static readonly Regex markdownLinkRegex = new(@"\[(.*?)\]((\(.*?\))|(\s*:\s*\w+))", RegexOptions.Compiled);
```

then have this as 

```suggestion
            markdownText = markdownLinkRegex .Replace(markdownText, "$1");
```